client/chat/sdk.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from typing import AsyncIterator, Dict
+
+import vertexai
+from google.cloud.aiplatform_v1beta1.types import content as gapic_content_types
+from vertexai.preview.generative_models import ChatSession as GenChatSession
+from vertexai.preview.generative_models import GenerationConfig, GenerativeModel
+from vertexai.preview.language_models import ChatModel
+from vertexai.preview.language_models import ChatSession as LangChatSession
+from vertexai.preview.language_models import CodeChatModel
+from vertexai.preview.language_models import (
+    CodeChatSession as LangCodeChatSession,
+)
+
+from aidial_adapter_vertexai.llm.vertex_ai_deployments import (
+    ChatCompletionDeployment,
+)
+from aidial_adapter_vertexai.universal_api.request import ModelParameters
+from aidial_adapter_vertexai.universal_api.token_usage import TokenUsage
+from client.chat.base import Chat
+
+log = logging.getLogger(__name__)
+
+
+HarmCategory = gapic_content_types.HarmCategory
+HarmBlockThreshold = gapic_content_types.SafetySetting.HarmBlockThreshold
+
+BLOCK_NONE_SAFETY_SETTINGS: Dict[HarmCategory, HarmBlockThreshold] = {
+    HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT: HarmBlockThreshold.BLOCK_NONE,
+    HarmCategory.HARM_CATEGORY_HATE_SPEECH: HarmBlockThreshold.BLOCK_NONE,
+    HarmCategory.HARM_CATEGORY_HARASSMENT: HarmBlockThreshold.BLOCK_NONE,
+    HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT: HarmBlockThreshold.BLOCK_NONE,
+}
+
+
+LangSession = LangChatSession | LangCodeChatSession
+
+
+class SDKLangChat(Chat):
```

Comment:
I think adding descriptions to these classes to explain their purpose would be useful.

Reply:
Added descriptions to the individual modules.